sys/auto_init/auto_init.c

```diff
@@ -90,6 +94,9 @@ void auto_init(void)
     config_load();
 #endif
 
+#ifdef MODULE_TINYMT32
+    genrand_init(0);
```

Comment:
Yes, that is intentional, and the same as with all other PRNG's we have. We need a centralized solution for this... My entropy collection PR goes in that direction. I also have a branch were I use the drift between xtimer and RTC to generate random bytes. But that should be solved centrally for all PRNGs. I just auto-init tinymt this way so it doesn't always return 0. The other PRNG's also have implicit but constant initial seeds.